index.html

```diff
@@ -155,10 +155,15 @@
 <a>Decentralized identifiers</a> (DIDs) are a new type of identifier that
 enables verifiable, decentralized digital identity. A <a>DID</a> identifies any
 subject (e.g., a person, organization, thing, data model, abstract entity, etc.)
-that the controller of the <a>DID</a> decides that it identifies. These new
-identifiers are designed to enable the controller of a <a>DID</a> to prove
-control over it and to be implemented independently of any centralized registry,
-identity provider, or certificate authority. <a>DID</a>s are URLs that associate
+that the controller of the <a>DID</a> decides that it identifies.
+
+In contrast to typical, federated identifiers, DIDs have been designed
+so that they may be implemented independently of any centralized
+registry, identity provider or certificate authority. Specifically,
+DIDs enable the controller of an identity alone to prove control over
+it without permission from any other party.
```

Comment:
Hmm, I've tried to help cover a few more cases and avoid more controversy with this rewording. Tell me what you think:

```suggestion
In contrast to typical, federated identifiers, DIDs have been designed
so that they may be decoupled from centralized registries, identity providers,
and certificate authorities. Specifically, while other parties may be used
to help enable the discovery of information related to a <a>DID</a>,
the design enables the controller of a <a>DID</a> to prove control over it
without requiring permission from any other party.
```

I would have liked to say "discovery of a DID Document" instead of "information related to a DID" -- but "DID Document" isn't introduced until the next paragraph. Unfortunately, I don't see an easy way to "put the key points up front" if we have to push the "DID Document" reference before them. Perhaps you can think of a way to simplify.

One point of contention here is that other "parties" (which may be decentralized ledgers or blockchains) are used, for many DID methods, to resolve a DID Document -- after which control can be proven independently by the controller. A key difference is that the controller proves control at that point using cryptography (keys that they control) vs. simply trusting some federated identity provider. However, unless the DID controller hands you the DID Document directly (some DID methods rely on this, others do not), you have to be able to resolve it using some system that isn't "just" the controller entity.

It is also the case that control can sometimes be better trusted with the combination of the controller AND a community of users (blockchain participants) backing the DID Document (plus the crypto behind a blockchain) vs. receiving a DID Document directly that may have been produced using compromised keys ... that have since been revoked on the aforementioned blockchain.